ldms/src/sampler/zfs_common/Makefile.am

```diff
@@ -0,0 +1,7 @@
+libzfs_common_helper_la_CPPFLAGS = @OVIS_INCLUDE_ABS@ \
+	      -I/usr/include/libspl \
+	      -I/usr/include/libzfs
+
+noinst_LTLIBRARIES = libzfs_common_helper.la
+libzfs_common_helper_la_SOURCES = zfs_common.c zfs_common.h
+libzfs_common_helper_la_LIBADD = $(CORE_LIBADD)
```

Comment:
This line should be removed too. There is no "CORE_LIBADD" in this file, so this line isn't doing anything.